src/lib/formValidator.js

```diff
@@ -112,7 +112,7 @@ const newProfileValidator = values => {
     err.lastName_error = 'Required';
   }
 
-  if (email === undefined) {
+  if (!email || email === undefined) {
```

Comment:
This is redundant now, `!email` will always cover the case where `email === undefined` (but not vice versa). You could just have `!email`

Reply:
Done. (removed email === undefined)